### PR TITLE
TOOLSDOC-610: Added text and two images to article

### PR DESCRIPTION
**Pull request by mishaone**
Minor addition and updates to content. @xcoulon, please have a look and merge if it looks OK.

### CODE REVIEW

File: documentation/howto/hmt_firstapp.adoc [hunk 1]
```diff
@@ -236,6 +238,98 @@ image::hmt_enter_push_details.png[Details of the Push Added in the Push Branch m
 .Cordova Application Published on the FeedHenry Server
 image::hmt_cordova_app_on_feedhenry.png[Cordova Application Published on the FeedHenry Server]
 
+[[icon_change]]
```
Comment (xcoulon) [inline documentation/howto/hmt_firstapp.adoc]: this should be "change_icon"

**Comment by xcoulon** [comment]
Just a minor issue with the anchor name for the new chapter. 
Also, may I suggest that you rename the level-3 sections to 4.1 and 4.2, because the small difference of font size between chapters and sub-chapters make the numbering a little confusing (feel free to ping me on IRC to discuss about it if this comment is not clear)

**Comment by mishaone** [comment]
@xcoulon should be OK now. Please have a look.

**Comment by xcoulon** [comment]
thanks for the updates, @mishaone !
I rebased the commits and merged them.

**Pull request closed without merging**